idb/postgres/postgres.go

```diff
@@ -223,7 +218,7 @@ func (db *IndexerDb) AddBlock(block *bookkeeping.Block) error {
 			proto.EnableAssetCloseAmount = true
```

Comment:
Do you think things like this could move into algod also?

Reply:
The main reason would be to keep consensus param "magic" as close to the source as possible. If we have an interface in algod designed for Indexer, shouldn't we use it?

Reply:
I don't have a strong opinion. @tsachiherman maybe does.